docs/tutorials/custom/custom-base-dog.md

```diff
@@ -64,11 +64,11 @@ If you choose to install the full Raspberry Pi OS (64 bit) including the desktop
 
 {{% /alert %}}
 
-1.  [Install `viam-server` and connect your robot to the Viam app](/get-started/installation/#install-viam-server).
+{{% snippet "setup.md" %}}
```

Comment:
Nit: Previous RPi Setup guide step is a single item, since the include can't be part of a numbered list. What do you think of un-numbering line 59?

Reply:
done